<?xml version="1.0"?>
<launch>

<arg name="rate"                      default="1"                               />
<arg name="cloud"                     default="/xtion/depth_registered/points"  />
<arg name="frame"                     default="/base_footprint"                 />
<arg name="passthrough_axis"          default="z"                               />
<arg name="passthrough_min"           default="0.4"                             />
<arg name="passthrough_max"           default="1.6"                             />
<arg name="downsampling_size"         default="0.01"                            />
<arg name="remove_points_below_plane" default="true"                            />

<node name="segment_table" pkg="tiago_pcl_tutorial" type="segment_table" output="screen">
  <remap from="cloud"                     to="$(arg cloud)"     />
  <param name="rate"                      type="double" value="$(arg rate)"  />
  <param name="frame"                     type="string" value="$(arg frame)"  />
  <param name="passthrough_axis"          type="string" value="$(arg passthrough_axis)"  />
  <param name="passthrough_min"           type="double" value="$(arg passthrough_min)"   />
  <param name="passthrough_max"           type="double" value="$(arg passthrough_max)"   />
  <param name="downsampling_size"         type="double" value="$(arg downsampling_size)" />
  <param name="remove_points_below_plane" type="bool"   value="$(arg remove_points_below_plane)" />
</node>

</launch>
